bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -607,7 +607,7 @@
 		<item-type>Switch</item-type>
 		<label>Steering Wheel Heater</label>
 		<description>Indicates if the steering wheel heater is switched on</description>
```

Comment:
Please keep the same description there, in the i18n file and preferably in documentation (README).
With your change, we have now 3 different descriptions.
I would suggest to keep the one you set in the i18n file, that is:  Turns On/Off the steering wheel heater